lib/screens/v2/widget_instance/elevator_status.ex

```diff
@@ -182,16 +188,18 @@ defmodule Screens.V2.WidgetInstance.ElevatorStatus do
          %Alert{effect: :elevator_closure, informed_entities: entities} = alert,
          %__MODULE__{now: now, stop_sequences: stop_sequences} = t
        ) do
-    stations = get_stations_from_entities(entities)
+    stations = for %{stop: stop} <- entities, do: stop
```

Comment:
My bad, this code will likely give us a list with a lot of `nil` elements because each entity always has a `stop` key (which is potentially `nil`). Feel free to confirm.

That's not going to break anything since we just use this list in equality checks, but it could slightly impact efficiency.

Suggestion:
```suggestion
    stations = for %{stop: stop} when is_binary(stop) <- entities, do: stop
```
or even more specific since we're only looking for platform stop IDs:
```suggestion
    informed_platforms = for %{stop: stop} when is_binary(stop) <- entities, match?({_n, ""}, Integer.parse(stop)), do: stop
```

Reply:
Wow, super cool. Elixir is awesome.